template/package.json

```diff
@@ -31,16 +31,21 @@
   "dependencies": {
     "@emotion/core": "^10.0.6",
     "@emotion/native": "^10.0.6",
+    "@react-native-community/masked-view": "^0.1.10",
+    "@react-navigation/bottom-tabs": "^5.6.1",
+    "@react-navigation/native": "^5.6.1",
+    "@react-navigation/stack": "^5.6.2",
     "emotion-theming": "^10.0.19",
     "react": "16.13.1",
     "react-native": "0.62.2",
     "react-native-elements": "^1.2.7",
-    "react-native-gesture-handler": "^1.1.0",
-    "react-native-safe-area-context": "^0.7.3",
+    "react-native-gesture-handler": "^1.6.1",
+    "react-native-reanimated": "^1.9.0",
+    "react-native-safe-area-context": "^3.0.7",
     "react-native-safe-area-view": "^1.0.0",
-    "react-native-screens": "^2.0.0-beta.2",
+    "react-native-screens": "^2.9.0",
     "react-native-splash-screen": "^3.1.1",
-    "react-native-vector-icons": "^6.6.0",
+    "react-native-vector-icons": "6.6.0",
```

Comment:
removed carat as v6.7.0 includes a bug.
will upgrade to v7 in another effort.

Reply:
Awesome, thanks for capturing this! Let's create an issue to address it.